crowbar_framework/app/controllers/api/crowbar_controller.rb

```diff
@@ -41,6 +41,7 @@ def upgrade
     end
   rescue Crowbar::Error::StartStepRunningError,
          Crowbar::Error::StartStepOrderError => e
+    log_exception(e)
```

Comment:
You're right. I initially added logging everywhere, then removed those which are control-flow-exceptions and must have missed this one.